lib/ecto/repo.ex

```diff
@@ -1651,6 +1651,8 @@ defmodule Ecto.Repo do
     * `:stale_error_message` - The message to add to the configured
       `:stale_error_field` when stale errors happen, defaults to "is stale".
 
+    * `:allow_stale` - Doesn't error if delete is stale. Defaults to `false`.
```

Comment:
```suggestion
    * `:allow_stale` - Doesn't error if insert is stale. Defaults to `false`.
      This may happen if there are rules or triggers in the database that
      reject the insert operation.
```